R/module_nested_tabs.R

```diff
@@ -160,6 +169,12 @@ srv_teal_module.teal_modules <- function(id,
   moduleServer(id = id, module = function(input, output, session) {
     logger::log_debug("srv_teal_module.teal_modules initializing the module { deparse1(modules$label) }.")
 
+    observeEvent(input$active_tab, once = TRUE, {
+      if (input$active_tab == "teal_data_module") {
+        setBookmarkExclude("active_tab")
+      }
+    })
```

Comment:
sorry I think I had a bad suggestion. I was thinking more about following, but this would be bad if there was a `teal_data_module`. I think previous implementation (one you commented out in module_teal.R) made more sense.

```
onBookmark({
  if (input$active_tab == "teal_data_module") {
        setBookmarkExclude("active_tab")
   }
})
```